scripts/bash/backendStartDev.sh

```diff
@@ -16,7 +16,7 @@ type_script=false
 inspect_port=${1}
 
 # Start server command for JS
-start_server="nodemon -w src --exec \"babel-node src --inspect=${inspect_port} --config-file '../../babel.config.json'\""
+start_server="nodemon -w src --exec \"babel-node src --inspect=${inspect_port} --stack-trace-limit=100 --stack-size=1024 --config-file '../../babel.config.json'\""
```

Comment:
The default number of stack trace is not large enough to provide informative logs as it contains some unnecessary track in node_modules (path to yup modules for example). 

We don't have to worry about the long and duplicated error stack in logs, because long error stack will be trimmed and pass into other servers that are called from.